dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10Jax/bootstrap.properties

```diff
@@ -1 +1,2 @@
-bootstrap.include=../testports.properties
\ No newline at end of file
+bootstrap.include=../testports.properties
+websphere.java.security.exempt=true
```

Comment:
Where we disable Java Security, we need a comment to say why. If it's a bug, there should be a link to the issue.